snowflake/assets/configuration/spec.yaml

```
@@ -0,0 +1,176 @@
+          - snowflake.pipe
+          - snowflake.replication
+        value:
+          type: array
+          items:
+            type: string
+          example:
+            - snowflake.query
+            - snowflake.billing
+            - snowflake.storage
+            - snowflake.logins
+      - name: custom_queries
+        description: |
+          Each query must have 2 fields:
+          1. query - The SQL to execute. It can be a simple statement or a multi-line script.
+                     Use the pipe `|` if you require a multi-line script.
+          2. columns - The list representing each column, ordered sequentially from left to right.
+                       The number of columns must equal the number of columns returned in the query.
+                       There are 2 required pieces of data:
+                         a. name - The suffix to append to `clickhouse.` to form
+                                   the full metric name. If `type` is `tag`, this column is
+                                   considered a tag and applied to every
+                                   metric collected by this particular query.
+                         b. type - The submission method (gauge, monotonic_count, etc.).
+                                   This can also be set to `tag` to tag each metric in the row
+                                   with the name and value of the item in this column. You can
+                                   use the `count` type to perform aggregation for queries that
+                                   return multiple rows with the same or no tags.
+                       Columns without a name are ignored, use this for any column you wish to skip:
+                         - {}
+          3. tags (optional) - A list of tags to apply to each metric.
+        enabled: false
+        value:
+          type: array
+          items:
+            type: object
+            properties:
+              - name: query
+                type: string
+              - name: columns
+                type: array
+                items:
+                  type: object
+              - name: tags
+                type: array
+                items:
+                  type: string
+          example:
+            - query: SELECT foo, COUNT(*) FROM table.events GROUP BY foo
+              columns:
+                - name: foo
+                  type: tag
+                - name: event.total
+                  type: gauge
+              tags:
+                - test:snowflake
+      - template: instances/tags
+      - template: instances/service
+      - template: instances/global
+        overrides:
+          min_collection_interval.description: |
+            This changes the collection interval of the check. For more information, see:
+            https://docs.datadoghq.com/developers/write_agent_check/#collection-interval
+
+            NOTE: The Snowflake ACCOUNT_USAGE schema views do not update in frequently.
```

```suggestion
            NOTE: The Snowflake ACCOUNT_USAGE schema views do not update very frequently.
```

In addition to that minor change, is there anything more specific about the update intervals we can share?